docs/source/_templates/layout.html

```diff
@@ -0,0 +1,10 @@
+{% extends "!layout.html" %}
+<link rel="canonical" href="{{ theme_canonical_url }}{{ pagename }}.html" />
+
+{% block footer %}
+{{ super() }}
+<script script type="text/javascript">
+  var collapsedSections = ['Best practices', 'Lightning API', 'Optional extensions', 'Tutorials', 'api', 'Bolts', 'Examples', 'Common Use Cases', 'Partner Domain Frameworks', 'Community'];
```

Comment:
I would rather feature Flash than Bolts :]

Reply:
Haha, perhaps can be dropped completely as we have this:
<img width="276" alt="Screenshot 2021-07-01 at 10 02 45" src="https://user-images.githubusercontent.com/9592806/124097534-8e54e080-da53-11eb-9a6c-0fd8d922210f.png">